packages/forklift-console-plugin/src/modules/Plans/views/create/PlanCreatePage.style.css

```diff
@@ -82,4 +82,8 @@
 
 .forklift--create-plan--wizard-appearance-order {
   z-index: 1;
+}
+
+.odf-backing-storage__selection--width{
```

Comment:
is it used anywhere? tried to look in the PR but couldn't find, maybe I'm missing something

Reply:
Not used and therefore removed